e2e/tests/stepDefinitions/manage/bookingReport.ts

```diff
@@ -3,7 +3,7 @@ import { Given, Then } from '@badeball/cypress-cucumber-preprocessor'
 import { Cas3ReportType } from '@approved-premises/api'
 import Page from '../../../../cypress_shared/pages/page'
 import DashboardPage from '../../../../cypress_shared/pages/temporary-accommodation/dashboardPage'
-import ReportNewPage from '../../../../cypress_shared/pages/temporary-accommodation/manage/reportNew'
+import ReportNewPage from '../../../../cypress_shared/pages/temporary-accommodation/manage/reportIndex'
```

Comment:
Can we get this one renamed too? Obviously will also need other usages in this file done also!

```suggestion
import ReportIndexPage from '../../../../cypress_shared/pages/temporary-accommodation/manage/reportIndex'
```